src/main/kotlin/app/revanced/patches/youtube/layout/sponsorblock/bytecode/patch/SponsorBlockBytecodePatch.kt

```diff
@@ -41,11 +40,11 @@ import org.jf.dexlib2.iface.reference.StringReference
 @Patch
 @DependsOn(
     dependencies = [
-        VideoInformationPatch::class, // updates video information and adds method to seek in video
+        IntegrationsPatch::class,
         VideoIdPatch::class,
+        VideoInformationPatch::class,   // video seek, get current playback speed
+        PlayerTypeHookPatch::class,     // checks if current video is a short
```

Comment:
The comments don't really explain why the dependencies are required.

What is video seek used for, and why is the current playback speed necessary?
Why is it necessary to check if the video is a short?

```suggestion
        VideoInformationPatch::class,   // video seek, get current playback speed
        PlayerTypeHookPatch::class,     // checks if the current video is a short
```